src/test/java/org/folio/rest/impl/InvoiceLinesApiTest.java

```diff
@@ -68,11 +81,43 @@ public void deleteInvoicingInvoiceLinesByIdTest() {
 
   @Test
   public void postInvoicingInvoiceLinesTest() throws Exception {
-    String jsonBody = getMockData(INVOICE_LINE_SAMPLE_PATH);
+    logger.info("=== Test create invoice line - 201 successfully created ===");
+    InvoiceLine reqData = getMockAsJson(INVOICE_LINE_SAMPLE_PATH).mapTo(InvoiceLine.class);
```

Comment:
looks like reqData is not being used anywhere, can be removed

Reply:
Removed